go/runtime/client/client.go

```diff
@@ -138,6 +144,12 @@ func (c *runtimeClient) SubmitTx(ctx context.Context, request *api.SubmitTxReque
 	}
 	c.Unlock()
 
+	req := &txnscheduler.SubmitTxRequest{
+		RuntimeID:     request.RuntimeID,
+		ExpectedEpoch: watcher.EpochLatest,
```

Comment:
This is racy, watcher runs in a separate goroutine. Make `EpochLatest` private and add accessor that performs synchronization.